app/src/main/java/com/brainwallet/ui/BrainwalletActivity.kt

```diff
@@ -115,6 +116,9 @@ class BrainwalletActivity : BRActivity() {
 
                             LEGACY_EFFECT_ON_PAPERKEY_PROVED -> {
                                 BRSharedPrefs.putPhraseWroteDown(this@BrainwalletActivity, true)
+                            }
+
+                            LEGACY_NAVIGATE_TO_TOP_UP -> {
```

Comment:
@andhikayuana hope you dont mind me fiddling with your work. Need to get this out for Investor review